api/serializers.py

```diff
@@ -4,11 +4,21 @@
 from django.contrib.auth.models import User
 
 
-class RelayAddressSerializer(serializers.ModelSerializer):
+class PremiumValidatorsMixin:
```

Comment:
Will you consider adding test for when the user tries to set the block_list_emails when they are not a premium user?

Reply:
Good call! Adding the tests uncovered a couple of fixes to make!